course/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Course
+{
+    class Course
+    {
+        static void PrintArrayElements(int[] arrayToPrint)
+        {
+            Array.ForEach(arrayToPrint, el => Console.WriteLine(el));
+        }
+
+        static int[] BubbleSorting(int[] arrayToSort)
+        {
+            for (int i = 0; i < arrayToSort.Length; i++)
+            {
+                for (int j = 0; j < arrayToSort.Length - 1; j++)
```

Comment:
you can iterate from 0 to **arrayToSort.Length - i**, do not need to go though whole array